setup.cfg

```diff
@@ -43,6 +43,9 @@ filterwarnings =
 env =
     ADMINS=Test Example <test@example.com>,Test2 <test2@example.com>
     LANGUAGES=en,de,fr
+    OIDC_OP_USER_ENDPOINT=mock://emeis.github.com/openid/userinfo
```

Comment:
Wow ok never seen that before. While it's a cool idea and definitely TIL, I'm not sure how good of an idea it is to hide this in the config somewhere. Wouldn't you rather wanna make a `oidc_settings` fixture which includes those settings?
Btw what does the `mock://` do, just for explicit documentation?

Reply:
This is the same as in our other projects (caluma, dms, ...). I'd rather not having an autouse fixture just for setting an env var.

For information about `mock://`, see the [requests-mock documentation](https://requests-mock.readthedocs.io/en/latest/).